Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:10
+# Use alpine node version
+FROM node:10.15.3-alpine
 
-# Create app directory
-WORKDIR /usr/local
+# Create working directory and set node user as owner
+WORKDIR /usr/src/app
```

Comment:
We agreed that this would be the location we'd deploy all services too as it's "standard".  (There isn't an actual standard but this is where a lot of the community seem to agree with)